conans/test/functional/build_helpers/cmake_flags_test.py

```diff
@@ -1,7 +1,9 @@
 import os
 import platform
+import pytest
```

Comment:
pytest is not a builtin python module, imports shouldn't be here (but not a big problem)

Reply:
right. I'll change them.